modules/ROOT/pages/cluster-create.adoc

```diff
@@ -5,53 +5,49 @@ endif::[]
 
 == Prerequisites
 
-Before creating a cluster, ensure you have performed the following:
+Before you can create a cluster, you must:
 
 * Install Mule version 3.6 or later, or API gateway runtime version 2.0 or later.
 * Install Runtime Manager Agent version 1.1.1 or later.
 * Create a server for each Mule you want to include in your cluster.
 
-Also, note the following considerations when creating a cluster:
-
-* None of the servers you select for creating a new cluster can contain any *previously deployed applications*.
-
-* Multicast servers can be in the status states of *Running* or *Disconnected*, whereas unicast servers can be only in the *Running* state. 
-
+[NOTE]
+--
+* None of the servers you select for a new cluster can contain any previously deployed applications.
```

Comment:
nor can they be members of another cluster/group. That'd be good to include here?